crates/uv-build-frontend/src/lib.rs

```diff
@@ -921,13 +921,15 @@ impl PythonRunner {
     ) -> Result<PythonRunnerOutput, Error> {
         /// Read lines from a reader and store them in a buffer.
         async fn read_from(
-            mut reader: tokio::io::Lines<tokio::io::BufReader<impl tokio::io::AsyncRead + Unpin>>,
+            mut reader: tokio::io::Split<tokio::io::BufReader<impl tokio::io::AsyncRead + Unpin>>,
             mut printer: Printer,
             buffer: &mut Vec<String>,
         ) -> io::Result<()> {
             loop {
-                match reader.next_line().await? {
-                    Some(line) => {
+                match reader.next_segment().await? {
+                    Some(line_buf) => {
+                        let line_buf = line_buf.strip_suffix(b"\r").unwrap_or(&line_buf);
+                        let line = String::from_utf8_lossy(line_buf).into();
```

Comment:
I do think `bstr` would be better for this, and would suggest using it (since we already depend on it, albeit transitively, in uv), but my memory here is that we need to write valid UTF-8. So while `bstr` wouldn't require this "lossy" step, I believe [writing to the underlying stderr would still require valid UTF-8 anyway](https://github.com/rust-lang/rust/blob/ce697f919d88b143d70312c0e7f3952fcb9869db/library/std/src/sys/pal/windows/stdio.rs#L102-L138). So `bstr` would just absolve you of needing to handle line iteration, but I think this is fine as-is.